scripts/netContain/Dockerfile

```diff
@@ -17,30 +17,18 @@
 
 FROM ubuntu:16.04
```

Comment:
Though this is not part of your change , Any specific reason on why are we using ubuntu image here ? It actually fattens up our final container image.